library/ajax/execute_background_services.php

```diff
@@ -37,10 +37,10 @@
  * Returns a count of due messages for current user.
  *
  * @package   OpenEMR
- * @link      http://www.open-emr.org
- * @author    EMR Direct <http://www.emrdirect.com/>
+ * @link      s://www.open-emr.org
```

Comment:
missing a couple letters here

Reply:
Thanks, @bradymiller. I found a few more http. I will revise this and push the changes shortly.

Reply:
Revised and fixed.